src/main/webapp/app/shared/layouts/navbar/navbar.component.ts

```diff
@@ -268,18 +270,38 @@ export class NavbarComponent implements OnInit, OnDestroy {
         plagiarism_cases: 'artemisApp.plagiarism.cases.pageTitle',
     };
 
+    studentPathBreadcrumbTranslations = {
+        exams: 'artemisApp.courseOverview.menu.exams',
+        exercises: 'artemisApp.courseOverview.menu.exercises',
+        lectures: 'artemisApp.courseOverview.menu.lectures',
+        learning_goals: 'artemisApp.courseOverview.menu.learningGoals',
+        statistics: 'artemisApp.courseOverview.menu.statistics',
+        discussion: 'artemisApp.metis.discussion.label',
```

Comment:
```suggestion
        discussion: 'artemisApp.metis.communication.label',
```
`discussion` is outdated

Reply:
Thanks, done in 4abeaa1